CSharp/Samples/Microsoft.Bot.Sample.FormFlowAttachmentsBot/Web.config

```diff
@@ -87,6 +87,10 @@
         <assemblyIdentity name="Microsoft.Azure.Documents.Client" publicKeyToken="31bf3856ad364e35" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-1.11.0.0" newVersion="1.11.0.0" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="System.Net.Http" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0" />
+      </dependentAssembly>
```

Comment:
risky?

Reply:
Package Manager did this. Not sure of the ramifications.

Reply:
SourceLink  creates no runtime dependencies or package dependencies that I know of. It's just that symbols have links to source in GitHub. Tracing during debug then automatically downloads the source as needed.

Reply:
Bot Builder V4 already has this implemented.

Reply:
The SourceLink.Create.CommandLine NuGet package gets imported in the project builds.